### PR TITLE
feat(qvst_result): Add qvst result on admin

### PR DESCRIPTION
**Pull request by NyraSama**
- Add ability to add/edit qvst result link
- Change pipeline to be able to manage qvst result link only when campaign is archived

### CODE REVIEW

File: lib/presentation/pages/qvst/content/qvst_content_stats.dart [hunk 1]
```diff
@@ -146,4 +161,50 @@ class QvstContentStats extends ConsumerWidget {
       ),
     );
   }
+
+  Future<void> updateApiCallAction({
```
Comment (PiotrFLEURY) [inline lib/presentation/pages/qvst/content/qvst_content_stats.dart:165]: This method should be renamed, what's the business goal ? not the technical one

**Review by Theo-lbg — APPROVED** [review]
LGTM

File: lib/presentation/widgets/qvst/qvst_status_of_campaign_and_button.dart [hunk 2]
```diff
@@ -71,12 +70,13 @@ class _QvstStatusOfCampaignAndButtonState
...
-            : (campaignStatus.toUpperCase() == QvstCampaignStatusEnum.open.name)
-                ? QvstCampaignStatusEnum.closed.name
-                : QvstCampaignStatusEnum.archived.name;
+    String statusUpdated = (stats.campaignStatus.toUpperCase() ==
+            QvstCampaignStatusEnum.draft.name)
+        ? QvstCampaignStatusEnum.open.name
+        : (stats.campaignStatus.toUpperCase() ==
+                QvstCampaignStatusEnum.open.name)
+            ? QvstCampaignStatusEnum.closed.name
+            : QvstCampaignStatusEnum.archived.name;
```
Comment (PiotrFLEURY) [inline lib/presentation/widgets/qvst/qvst_status_of_campaign_and_button.dart:79]: hard to read and understand, can you simplify ?

**Review by PiotrFLEURY — COMMENTED** [review]
Please consider comments

**Review by PiotrFLEURY — APPROVED** [review]
LGTM

**Pull request merged**